src/main/java/calculator/CalculatorMain.java

```diff
@@ -0,0 +1,21 @@
+package calculator;
+
+public class CalculatorMain {
+    public static void main(String[] args) {
+        String inputFilePath = "src/main/resources/input.txt";
+        String outputFilePath = "src/main/resources/output.txt";
+
+        GetDataFromInputFile getDataFromInputFile = new GetDataFromInputFile(inputFilePath);
+        Calculation result = new Calculation();
+        PutResultIntoOutputFile putResultIntoOutputFile = new PutResultIntoOutputFile(outputFilePath);
+
+        getDataFromInputFile.readDataFromFile();
+
+        System.out.println(getDataFromInputFile.getOperand1() + " "
```

Comment:
looks like there's some kind of a mess inside sout statement;
it might be a good idea separate calc logic and logging....however business logic is preferred before logging so it should be on the top but not inside log statements.

Reply:
Local variables were added in daaeb4b44ce00b5dbd8bee93a02578f4c0921046.